src/platform_impl/web/web_sys/event.rs

```diff
@@ -5,10 +5,13 @@ use std::convert::TryInto;
 use web_sys::{HtmlCanvasElement, KeyboardEvent, MouseEvent, PointerEvent, WheelEvent};
 
 pub fn mouse_button(event: &MouseEvent) -> MouseButton {
+    // https://w3c.github.io/uievents/#dom-mouseevent-button
```

Comment:
```suggestion
    // https://www.w3.org/TR/uievents/#dom-mouseevent-button
```
We should link to the W3C instead of the editors draft.